<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Seller</class>
 <widget class="QWidget" name="Seller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="seller_listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="seller_back">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="sum">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>总库存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seller_sell">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>销售</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seller_find">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="find_input">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="seller_refresh">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
